code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -308,8 +310,8 @@ em						{font-style: normal;	font-weight: bold;}
 .rose					{color: #ff5050;}
 .info					{color: #9ab0ff;}
 .notice					{color: #6685f5;}
-.tinynotice				{color: #6685f5; font-style: italic;	font-size: 85%;}
-.smallnotice			{color: #6685f5; font-style: italic;	font-size: 90%;}
+.tinynotice				{color: #6685f5;	font-size: 85%;}
```

Comment:
I see you removed italics here and add it to when it's used.
Make a second one with italics and use it there.